src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -48,12 +48,12 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="NuGet.Core">
-      <HintPath>$(NuGetPackageRoot)\nuget.core\$(NuGetCoreV2Version)\lib\net40-Client\NuGet.Core.dll</HintPath>
+      <HintPath>$(PkgNuGet_Core)\lib\net40-Client\NuGet.Core.dll</HintPath>
       <Aliases>CoreV2</Aliases>
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <PackageDownload Include="NuGet.Core" />
+    <PackageReference Include="NuGet.Core" IncludeAssets="None" PrivateAssets="All" GeneratePathProperty="true" />
```

Comment:
```suggestion
    <PackageReference Include="NuGet.Core" ExcludeAssets="All" GeneratePathProperty="true" />
```

Reply:
I'm not 100% sure what the implementation is, but to me `IncludeAssets` means "use in current project and flow transitively", `PrivateAssets` means "use in current project, but don't flow transitively", and `ExcludeAssets means "don't use in current project, or flow transitively".

So no, I don't believe that `PrivateAssets` are needed.

Reply:
NuGet.Commandline is a special package that suppresses all dependencies, https://github.com/NuGet/NuGet.Client/blob/a487c8336b29a0e53dc46f4aab424fdd5de50e72/src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj#L20.